keyboards/kbdfans/kbd67/mkiirgbv3/rules.mk

```diff
@@ -0,0 +1,34 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = lufa-ms
```

Comment:
I'm concerned about this, actually. 

the mass storage bootloader is very, very prone to accidental bricking, as the hidden/system files that Windows and MacOS can place can cause the bootloader to brick the device. 

It may be convenient to use, but it is a VERY BAD IDEA.  And the main reason that QMK supports it at all, is because there are boards that already have it on them.